app/code/Magento/Customer/Model/Session.php

```diff
@@ -152,18 +157,20 @@ public function __construct(
         \Magento\Framework\App\Http\Context $httpContext,
         CustomerRepositoryInterface $customerRepository,
         GroupManagementInterface $groupManagement,
-        \Magento\Framework\App\Response\Http $response
+        \Magento\Framework\App\Response\Http $response,
+        AccountConfirmation $accountConfirmation = null
     ) {
-        $this->_coreUrl = $coreUrl;
-        $this->_customerUrl = $customerUrl;
         $this->_configShare = $configShare;
+        $this->_coreUrl = $coreUrl;
```

Comment:
Could you revert all not related changes? Originally it was sorted in the same way as constructor arguments, it was logically correct

Reply:
**done**